src/lsdb/catalog/catalog.py

```diff
@@ -66,6 +67,17 @@ def get_ordered_healpix_pixels(self) -> List[HealpixPixel]:
         pixels = self.get_healpix_pixels()
         return np.array(pixels)[get_pixel_argsort(pixels)]
 
+    def head(self, n: int = 5) -> pd.DataFrame:
+        """Returns a few rows of data for previewing purposes.
+
+        Args:
+            n (int): The number of desired rows.
+
+        Returns:
+            A pandas DataFrame with up to `n` of data.
+        """
+        return self._ddf.head(n, npartitions=-1)
```

Comment:
It looks like `npartitions=-1` makes Dask loading all partitions, which is very slow. We can go smarter and optimize it for small `n`, sequentially examining partitions until we collect `n` rows.

Reply:
@hombit thanks for the feedback, you're right! I've applied a fix.